src/pages/app/posting_project_user/styles/BeautifyProfileStyles.css

```diff
@@ -39,7 +39,6 @@
     border:2px solid black;
     border-radius: 7px;
 }
-<<<<<<< HEAD
 .avatar-box{
     border:2px solid black;
     border-radius: 5px;
```

Comment:
The whole style is already separated for the `<ImagePreview/>` component
Please remove this style
If you want to use the image preview feature, just import it from the **component** folder

Reply:
ok got it